Makefile

```diff
@@ -113,12 +113,22 @@ endif
 	find . -name \*~ -delete
 	find . -name \#\* -delete
 	rm -f bin/crio
+	rm -f bin/crio.cross.*
 	make -C conmon clean
 	make -C pause clean
 	rm -f test/bin2img/bin2img
 	rm -f test/copyimg/copyimg
 	rm -f test/checkseccomp/checkseccomp
 
+
+local-cross: bin/crio.cross.windows.amd64 bin/crio.cross.darwin.amd64 bin/crio.cross.linux.amd64
+
+bin/crio.cross.%:
+	@echo "==> building $@"; \
```

Comment:
Do we need this echo?  The makefile will print the recipe lines as it launches them, and I think:

    GOOS=… GOARCH=… go build …

is a clear enough indicator on its own.